### PR TITLE
Update readme to direct users to v2

### PR DESCRIPTION
**Pull request by admah**
DO NOT MERGE until other deprecation tasks are complete.

### CODE REVIEW

**Comment by JacobMGEvans** [comment]
Oh that audit one is interesting.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # 🤠 wrangler
 
-**Note:** If you're looking for the latest version of the `wrangler` package, visit https://github.com/cloudflare/wrangler2
+**Note:** This repository contains version 1.x of wrangler, and is no longer updated. Continued development is ongoing at https://github.com/cloudflare/wrangler2.
```
Comment (penalosa) [inline README.md:3]:
```suggestion
**Note:** This repository contains version 1.x of wrangler, and is no longer updated. Continued development is ongoing at https://github.com/cloudflare/wrangler.
```

Comment (penalosa) [reply README.md:3]: (Once the repos have been renamed)

**Review by penalosa — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (JacobMGEvans) [reply README.md:3]: Good call!

**Pull request closed without merging**